pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go

```diff
@@ -61,24 +73,52 @@ func (m *mockStreamClient) GetTopology(
 
 // ConsumePartition implements the Client interface.
 func (m *mockStreamClient) ConsumePartition(
-	_ context.Context, address streamingccl.PartitionAddress, _ hlc.Timestamp,
+	ctx context.Context, address streamingccl.PartitionAddress, _ hlc.Timestamp,
 ) (chan streamingccl.Event, chan error, error) {
 	var events []streamingccl.Event
 	var ok bool
 	if events, ok = m.partitionEvents[address]; !ok {
 		return nil, nil, errors.Newf("no events found for paritition %s", address)
 	}
 
-	eventCh := make(chan streamingccl.Event, len(events))
+	eventCh := make(chan streamingccl.Event)
+	errCh := make(chan error)
 
-	for _, event := range events {
-		eventCh <- event
-	}
-	close(eventCh)
+	go func() {
+		defer close(eventCh)
```

Comment:
@annezhu98 one nit, though you can pick it up in your next PR to prevent another push since this is only testing code. We should close(errCh) too.